include/ocpp/common/websocket/websocket_uri.hpp

```diff
@@ -53,10 +54,10 @@ class Uri {
         return uri.str();
     }
 
-    websocketpp::uri get_websocketpp_uri() { // FIXME: wrap needed `websocketpp:uri` functionality inside `Uri`
-        return websocketpp::uri(this->secure, this->host, this->port,
-                                this->path_without_chargepoint_id /* is normalized with ending slash */ +
-                                    this->chargepoint_id);
+    websocketpp_utils::uri get_websocketpp_uri() { // FIXME: wrap needed `websocketpp:uri` functionality inside `Uri`
```

Comment:
Is that comment still relevant?

Reply:
I think in the future we should do it manually, without any 3rd party, so in that case it would still be relevant.